docs/samples/browser-plugin-meetings/app.js

```diff
@@ -2694,6 +2694,39 @@ function viewBreakouts(event) {
 
     return button;
   };
+  const createAskForHelpButton = (breakoutSession) => {
+    const button = document.createElement('button');
+
+    button.innerText = 'Help';
+
+    button.onclick = () => {
+      breakoutSession.askForHelp();
+    };
+
+    return button;
+  };
+  const createAskAllReturnButton = (breakoutSession) => {
+    const button = document.createElement('button');
+
+    button.innerText = 'Return';
```

Comment:
suggestion: I think something like "Ask to return" would be clearer than just "Return" here as that sounds very similar to Leave.

Reply:
Done for it